pkg/util/utils.go

```diff
@@ -71,7 +69,7 @@ func Retry(attempts int, callback func() error) (err error) {
 	return RetryAfter(attempts, callback, 0)
 }
 
-func ValidateProxyURI(uri string) bool {
+func ValidProxyURI(uri string) bool {
 	return govalidator.IsURL(uri)
```

Comment:
It seems some tools imply 1080 (curl for example). Also one might use the default port 80. Looking at the implementation of Transport in Go, the port stays completely opaque. If a proxy url is specified, it just tries to connect to this url.

That said, this does not have to stop us from adding constraints and enforcing a port to be specified. Just not sure whether it makes sense.